sdk/eventhub/event-hubs/review/event-hubs.api.md

```diff
@@ -232,7 +232,7 @@ export interface SendBatchOptions extends OperationOptions {
 }
 
 // @public
-export interface SubscribeOptions {
+export interface SubscribeOptions extends OperationOptions {
```

Comment:
This will also pull in the abortSignal which we don't use with `subscribe` today since it doesn't return a promise. Should we only pull in the tracing options?

Reply:
Good point, I'll split it out.